kea-vscode/src/views/pull-request-contents/commits/commits-root-tree-node.ts

```diff
@@ -39,4 +94,51 @@ export class CommitsRootTreeNode implements IParentTreeNode<CommitTreeNode> {
 
     return commits.map((commit) => new CommitTreeNode(this.#repository, commit));
```

Comment:
If a `CommitTreeNode` is for a remote commit, then it should have an icon indicating this.